server/newrelic.js

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable camelcase */
+'use strict';
+/**
+ * New Relic agent configuration.
+ *
+ * See lib/config/default.js in the agent distribution for a more complete
+ * description of configuration variables and their potential values.
+ */
+exports.config = {
+  /**
+   * Array of application names.
+   */
+  app_name: ['SDCServer'],
+  /**
+   * Your New Relic license key.
+   */
+  license_key: 'f504babc10a4424c2f380c87ce023c61b54dNRAL',

```

Comment:
I think you'll want to get this file removed from the git cache and then gitignore it, as your New Relic API Key exists here in plain text. Alternatively, abstract this key away into an environment variable that is then used by this file

Reply:
Good catch, I will remove and amend this issue